src/test/java/edu/kis/powp/jobs2d/events/SelectCopySecretCommand.java

```diff
@@ -0,0 +1,80 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.command.*;
+import edu.kis.powp.jobs2d.command.manager.DriverCommandManager;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+public class SelectCopySecretCommand implements ActionListener {
+
+	private static ICompoundCommand compound = new ICompoundCommand() {
+
+		List<DriverCommand> driverCommands = null;
+
+		@Override
+		public void execute(Job2dDriver driver) {
+			driverCommands.forEach((c) -> c.execute(driver));
+		}
+
+		@Override
+		public Iterator<DriverCommand> iterator() {
+			return driverCommands.iterator();
+		}
+
+		@Override
+		public DriverCommand clone() throws CloneNotSupportedException {
+			return (DriverCommand) super.clone();
+		}
+
+		@Override
+		public void setCommands(List<DriverCommand> driverCommands) {
+			this.driverCommands = driverCommands;
+		}
+	};
```

Comment:
Maybe it will be easier to create a separate implementation of ICompoundCommand in commands hierarchy.

Reply:
Maybe, but creation of implementation of ICompundCommand is a task given to another team.

Reply:
Yes, but they have to implement a specific ICompoundCommand (immutable). You can always add your own independent implementation.